gbus/abstractions.go

```diff
@@ -30,6 +30,7 @@ type BusConfiguration struct {
 type Bus interface {
 	HandlerRegister
 	Deadlettering
+	RawMessageHandling
```

Comment:
Wouldn't a chain design with default chain components would be more effective here and more clear?
I mean that at this point it seems to allow a lot of confusion in its capability while if we document the chain that would be generated and used if we do `RegisterSaga` or even the `HandleMessage` and `HandleEvent`...

Reply:
Perhaps, we can explore this design idea